iidm/iidm-test/src/main/java/com/powsybl/iidm/network/test/FourSubstationsNodeBreakerFactory.java

```diff
@@ -0,0 +1,623 @@
+/**
+ * Copyright (c) 2020, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.test;
+
+import com.powsybl.iidm.network.*;
+import org.joda.time.DateTime;
+
+import java.util.Objects;
+
+/**
+ * This test network is constituted of four substations, with five voltage levels.
+ * Below is a diagram of the network:
+ * <div>
+ *    <object data="doc-files/fourSubstationsNetwork.svg" type="image/svg+xml"></object>
+ * </div>
+ *
+ *
+ * @author Agnes Leroy <agnes.leroy at rte-france.com>
+ */
+
+public final class FourSubstationsNodeBreakerFactory {
+
+    private FourSubstationsNodeBreakerFactory() {
+    }
+
+    public static Network create() {
+        return create(NetworkFactory.findDefault());
+    }
+
+    public static Network create(NetworkFactory networkFactory) {
+        Objects.requireNonNull(networkFactory);
+
+        Network network = networkFactory.createNetwork("fourSubstations", "test");
+        network.setCaseDate(DateTime.parse("2017-06-25T17:43:00.000+01:00"));
+        network.setForecastDistance(0);
+
+        Substation s1 = network.newSubstation()
+                .setId("S1")
+                .setCountry(Country.FR)
+                .add();
+        VoltageLevel s1vl1 = s1.newVoltageLevel()
+                .setId("S1VL1")
+                .setNominalV(225.0)
+                .setLowVoltageLimit(0.0)
```

Comment:
It's weird to initialize only the low voltage limit. Initialize both voltage limits (80% and 120% are the common values), or none.